testing/load-aware/config.yaml

```diff
@@ -89,23 +87,13 @@ clusters:
     compute:
       dedicated: true
       machineset:
-        name: rhods-pipelines
-        count: null
+        name: workload-pods
+        count: 3
```

Comment:
can you make this `1` in the default configuration, and override with a higher node count in specific tests

maybe also, force it to `1` in the light test (`clusters.sutest.compute.machineset.count: 1`) so that your light test never runs with more than 1 node

... or ignore the first comment, only update the `light` profile, to have `3` as a default for all the `load-aware-e2e` tests 🤔 
as you prefer !

Reply:
Can do, I'll probably update the light profile